src/App.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { Login } from './pages/Login';
+import { Wallet } from './pages/Wallet';
 
 function App() {
-  return <div>Hello, TrybeWallet!</div>;
+  return (
+    <div>
+      <Switch>
+        <Route exact to="/" component={ Login } />
+        <Route to="/carteira" component={ Wallet } />
```

Comment:
Aqui estava uma das partes que faziam a aplicação funcionar incorretamente. O correto é usar `path`. O `to` é p/ componentes como `Redirect` e `Link`. Fazendo assim a sua aplicação deve voltar a renderizar as páginas corretamente. :grin: 

```suggestion
        <Route exact path="/" component={ Login } />
        <Route path="/carteira" component={ Wallet } />
```